composer.json

```diff
@@ -45,7 +44,6 @@
 		"symfony/polyfill-php80": "^1.23",
 		"symfony/polyfill-php81": "^1.27",
 		"symfony/process": "^5.4.3",
-		"symfony/service-contracts": "^2.5.0",
```

Comment:
This is going to install `3.4.1` which is `php >= 8.1`... Besides relying on CI which detects these mistakes I also manually check what versions these packages depend on...

Reply:
Ok, I'll revert that too. 

It must be hard for you to maintain it like this :) Quick check of composer.lock reveals few more:

- symfony/deprecation-contracts: >=8.1
- composer/pcre: ^7.4 || ^8.0
- psr/container: >=7.4.0

Maybe you want to pin those too.